### ISSUE TITLE
icns and ico files

### ISSUE THREAD

**Issue by skinnn** [skinnn/caffe-manager#5]
@reallinfo Can you create icns and ico files too? That would be necessary for mac and win builds. Thanks!

**Comment by reallinfo** [comment]
Sure, I will try to send it to you today.

**Comment by reallinfo** [comment]
can you take a look here? if these are the files you want, I will add them to PR.

https://drive.google.com/open?id=1xrZ_ELX6BtO8D732X10dHnXqGf2YQJUt

**Comment by skinnn** [comment]
Yes that's it. I just need the size 256 x 256 px . Also if you can, leave me some kind of contact ( skype if possible ) so I can get to you or recommend you to someone in case there is some design/work to be done in the future. Thanks.

**Comment by reallinfo** [comment]
Yes, i added. unfortunately I do not use skype. Is it possible to contact me from mail address?

komiserback@gmail.com

**Comment by skinnn** [comment]
@reallinfo Sure. I checked the pr but I can't find the .icns file. If you want you can export the 256 x 256 icns file, place it in a zip file and attach it here.
